share/toolchain-nxdk.cmake

```diff
@@ -21,10 +21,29 @@ set(NXDK 1)
 set(CMAKE_C_COMPILER "${NXDK_DIR}/bin/${TOOLCHAIN_PREFIX}-cc")
 set(CMAKE_C_STANDARD_LIBRARIES "${NXDK_DIR}/lib/libwinapi.lib ${NXDK_DIR}/lib/xboxkrnl/libxboxkrnl.lib ${NXDK_DIR}/lib/libxboxrt.lib  ${NXDK_DIR}/lib/libpdclib.lib ${NXDK_DIR}/lib/libnxdk_hal.lib ${NXDK_DIR}/lib/libnxdk.lib ${NXDK_DIR}/lib/nxdk_usb.lib") #"${CMAKE_CXX_STANDARD_LIBRARIES_INIT}"
 set(CMAKE_C_LINK_EXECUTABLE "${NXDK_DIR}/bin/${TOOLCHAIN_PREFIX}-link <FLAGS> <CMAKE_C_LINK_FLAGS> <LINK_FLAGS> <OBJECTS> -out:<TARGET> <LINK_LIBRARIES>")
+# Keep in sync with include paths in bin/nxdk-cc
+set(CMAKE_C_STANDARD_INCLUDE_DIRECTORIES
+        "${NXDK_DIR}/lib"
+        "${NXDK_DIR}/lib/xboxrt/libc_extensions"
+        "${NXDK_DIR}/lib/pdclib/include"
+        "${NXDK_DIR}/lib/pdclib/platform/xbox/include"
+        "${NXDK_DIR}/lib/winapi"
+        "${NXDK_DIR}/lib/xboxrt/vcruntime"
```

Comment:
indent size (8 space width) doesn't match with other indent size (4 space width)
The same for CMAKE_CXX_STANDARD_INCLUDE_DIRECTORIES setter line.

Reply:
The other indents are inside blocks, these are multiline continuations of a directive.

Reply:
In C/C++ indent style that I'm familiar with are usually align with:
&nbsp;&nbsp;&nbsp;&nbsp;A. Start after function's name and parentheses brace.
&nbsp;&nbsp;&nbsp;&nbsp;B. One tab indent width or same amount of spaces for indent width.

Except in the format you did, both format styles doesn't match. So, it should be only one indent width inside of a function if you are going for B style. If you had reviewed [Kitware's CMakeLists.txt](https://gitlab.kitware.com/cmake/cmake/-/blob/master/CMakeLists.txt) file, they use B style format. As for your format style, I am not familiar nor never seen it happen in other repositories. This includes NevolutionX and LithiumX's repositories. So, your format with extra indent width is new and unfamiliar.

I hope this is clear understanding and how it was missed from review remark.

Reply:
Double indent for continuation lines is used by at least one very large software company that I know of: https://google.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions.

That said, you are more than welcome to do a PR to unindent. Given that there is no prevailing example of CMakeLists.txt style in this repo I defer to the maintainers to decide on what color to paint the bikeshed. Ideally trivial indentation choices should just be enforced via automation, doubled continuation indent happens to be how my CLion instance auto-formatted these lines.

Reply:
Thanks for the mention of another example to support your case. Quite interesting how google does their code style formats.

What's also interesting is vscode showed me two spaces indent width for four spaces indent width in cmake files. Perhaps there should be some research how to make cmake files to enforce project preferred code style format. For now, I'll dismiss my review remark and have it fix later.